rasa_nlu/extractors/crf_entity_extractor.py

```diff
@@ -414,9 +415,15 @@ def _sentence_to_features(self, sentence):
                     prefix = prefixes[f_i_from_zero]
                     features = configured_features[f_i_from_zero]
                     for feature in features:
-                        # append each feature to a feature vector
-                        value = self.function_dict[feature](word)
-                        word_features[prefix + ":" + feature] = value
+                        if feature == "pattern":
+                            # add all regexes as a feature with form { name : is_matced }
+                            regex_patterns = self.function_dict[feature](word)
+                            for pattern_name, matched in regex_patterns.items():
+                                word_features[prefix + ":" + pattern_name] = matched
```

Comment:
`pattern_name` should be prefixed with the feature name to avoid collisions with other features, `prefix + ":" + feature + ":" + pattern_name`

Reply:
fixed